cli/codeFormat.js

```diff
@@ -0,0 +1,29 @@
+const prettier = require('prettier');
```

Comment:
move this file to `generators/client/templates/angular/webpack/_prettier.js` as a template and add an entry for this in the `generators/client/files-angular.js` file so that this file is generated during app creation.

Reply:
add "prettier": "1.9.2" as dependency to the generated apps package.json